.travis.yml

```diff
@@ -33,13 +35,13 @@ install:
   - pip install coverage
   - pip install coveralls
   - pip install Pillow
+  - pip install numpy
   - pip install .
 
 
 script:
    - cd test
-   - coverage run --source jpegenc -m py.test -s
-   - cp .coverage ../
```

Comment:
We want to merge this back to the main repository, your test should run with all the others (leave the original line).  In your local you can run just the tests you want (to save time) by using the command you added.